clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -376,11 +384,24 @@ private void logAll() {
         log.info(b.toString());
     }
 
+    public void logUnusedAndUnknown() {
```

Comment:
This class is public and hence a kip is required for this https://javadoc.io/doc/org.apache.kafka/kafka-clients/latest/org/apache/kafka/common/config/AbstractConfig.html

Reply:
Oh... Thanks for the reminder. @RivenSun2 , could you summit a small KIP for this change? Thanks.

Reply:
Did we consider the fact that we don't know the config names for pluggable components?

Reply:
TBH, no, I've never considered that! Thanks for the reminder. 

@RivenSun2 , sorry, I forgot the `AbstractConfig` is a public API. But since the v3.2.0 KIP freeze has passed, and I think this KIP might need more discussion, could you submit another PR to revert this change first? After KIP discussion/voting passed, we can start to work on it again. If you don't have time to submit a revert PR within this week, please let me know. Thank you.